src/app/components/homepage-components/navbar/input-field/input-field.component.scss

```diff
@@ -1,15 +1,81 @@
+@import "../../../../../mixins";
+
 .main-search-bar {
+  @include _600 {
+    height: 55px;
+    justify-content: center;
+    padding: 0 45px;
+  }
+  @include _420 {
+    justify-content: center;
+    height: 40px;
+    padding: 0 15px;
+  }
```

Comment:
you dont need justify-content inside _420 media, it already works from _600

Reply:
fixed